### DISCUSSION TITLE
transformer.jar有什么用

### DISCUSSION THREAD

**Discussion by mc-124** [Meloong-Git/PCL#3159] [提问] [answered]
\

**Comment by allMagicNB** [comment]
变压器，变形金刚（

**Reply by mc-124** [reply]
说真的啊

**Reply by allMagicNB** [reply]
> 说真的啊

应该是提供转换什么的作用

**Comment by zkitefly** [comment, marked as answer]
来自 https://github.com/HMCL-dev/HMCL/tree/javafx/minecraft%2Flibraries ，该 jar 的用途只是用来解决 1.13 optifine 和 forge 同时安装的问题，pcl 内置是当检测的有这个目录时放在这里，防止没这个启动不了（应该是这样
